Tests/Unit/NetworkServer/LoRaDeviceTest.cs

```diff
@@ -107,11 +107,12 @@ public async Task After_Saving_Frame_Counter_Changes_Should_Not_Have_Pending_Cha
         public async Task When_Initialized_New_OTAA_Device_Should_Have_All_Properties()
         {
             var appKey = TestKeys.CreateAppKey(0xABC0200000000000, 0x09);
+            var joinEui = new JoinEui(0xABC0200000000009);
 
             var twin = TestUtils.CreateTwin(
                 desired: new Dictionary<string, object>
                 {
-                    { "AppEUI", "ABC0200000000009" },
+                    { "AppEUI", joinEui.ToString() },
```

Comment:
Shouldn't this be using the `N` format to mean the same thing?

Reply:
Valid point, done in fd8bc7daed7f63f42291c59ed33ae9bcb17917e3